site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1650,14 +1659,12 @@ public function getAllSectionsForGradeable($gradeable) {
                 SELECT * FROM sections_registration
                 ORDER BY SUBSTRING(sections_registration_id, '^[^0-9]*'),
                 COALESCE(SUBSTRING(sections_registration_id, '[0-9]+')::INT, -1),
-                SUBSTRING(sections_registration_id, '[^0-9]*$')"
-            );
+                SUBSTRING(sections_registration_id, '[^0-9]*$')");
```

Comment:
It seems this closing parenthesis should be on the next line, but the string starts with a newline. Would it make more sense to take out the new line and let the parenthesis be on a line by itself?

Reply:
Sort of. I think the best course of action would be to put the query (and all long multiline queries) into a variable above the function call and define it as a heredoc instead of a multiline string. This would I think give the best combination of readability and formatting by phpcs.

Reply:
For a broader point, We need to codify exactly how we wish to see long queries written into a style document and then have it be something we properly follow in code-reviews, which defines how each block of a SQL query should be indented compared to the previous line. 

I will open an issue to continue that there, and then after we get consensus on agreed style, we can go through the entire file and reformat all queries as appropriate.